src/plugins/intel_gpu/src/graph/graph_optimizer/mark_runtime_skippable_nodes.cpp

```diff
@@ -47,6 +47,7 @@ void mark_runtime_skippable_nodes::run(program& p) {
             !node->is_type<non_max_suppression_gather>()) {
             // always to skip, no runtime execution
             node->can_be_optimized(true);
+            node->set_runtime_skippable(true);
```

Comment:
As I mentioned offline, this is against the original intention of this code block. 
Instead, I think we should fix the do_runtime_skip_xx functions to just return when the node is not "runtime_skippable"

Reply:
I see. I removed this code and used is_runtime_skippable instead of can_be_optimized in do_runtime_skip_xxx.
please review it again. thanks.